sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -168,7 +168,6 @@ public virtual Response<string> GetDigitalTwin(string digitalTwinId, Cancellatio
         /// <param name="digitalTwin">The application/json digital twin to create.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The created application/json digital twin and the http response.</returns>
-        /// <remarks>The digital twin must be the serialization of an instance of <see cref="Serialization.BasicDigitalTwin"/> or the serialization of an extension of that type.</remarks>
```

Comment:
We don't want to refer to the optional serialization types in the client documentation, and it is not true that it has to be a serialization of that or a derived type. It just has to match the DTDL schema of the model, and the other required fields for a digital twin, that happened to be specified in the BasicDigitalTwin.

Reply:
Are we referencing `BasicDigitalTwin` anywhere in our docs?

Reply:
Not in the DigitalTwinsClient API surface docs.

We'll refer to it in samples, and on the BasicDigitalTwin web docs.

The reason is, we can't have System.Text.Json types in our official API surface. I've gotten the Azure SDK team to give us the leeway to have these types available as optional.